File: 백준/Bronze/28691. 정보보호학부 동아리 소개/정보보호학부 동아리 소개.cs
using System;

class Q28691
{
    static void Main(string[] args)
    {
        string S = Console.ReadLine();
        
        if (S.Equals("M"))
        {
            Console.WriteLine("MatKor");
        }
        else if (S.Equals("W"))
        {
            Console.WriteLine("WiCys");
        }
        else if (S.Equals("C"))
        {
            Console.WriteLine("CyKor");
        }
        else if (S.Equals("A"))
        {
            Console.WriteLine("AlKor");
        }
        else if (S.Equals("$"))
        {
            Console.WriteLine("$clear");
        }
    }
}

File: 백준/Bronze/28691. 정보보호학부 동아리 소개/정보보호학부 동아리 소개.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        String S = sc.next();
        
        if (S.equals("M")) {
            System.out.println("MatKor");
        } else if (S.equals("W")) {
            System.out.println("WiCys");
        } else if (S.equals("C")) {
            System.out.println("CyKor");
        } else if (S.equals("A")) {
            System.out.println("AlKor");
        } else if (S.equals("$")) {
            System.out.println("$clear");
        }
    }
}